core/src/test/java/org/infinispan/persistence/BaseStoreFunctionalTest.java

```diff
@@ -275,6 +280,25 @@ public void writeToAllNonTxStores(MarshalledEntry marshalledEntry, AccessMode mo
       }
    }
 
+   public void testPutAllBatch() throws Exception {
+      int numberOfEntries = 100;
+      String cacheName = "testPutAllBatch";
+      ConfigurationBuilder cb = new ConfigurationBuilder();
+      cb.read(cacheManager.getDefaultCacheConfiguration());
+      createCacheStoreConfig(cb.persistence(), false);
+      cacheManager.defineConfiguration(cacheName, cb.build());
+
+      Cache<Integer, String> cache = cacheManager.getCache(cacheName);
+      Map<Integer, String> entriesMap = IntStream.range(0, numberOfEntries).boxed()
```

Comment:
We should also update `testRemoveCacheWithPassivation` so that it checks the batch method as well.